static/src/stylesheets/module/atoms/_storyquestions.scss

```diff
@@ -0,0 +1,139 @@
+.user__question {
+    padding-bottom: $gs-baseline;
+    overflow: auto;
+}
+.user__question-title {
+    font-weight: 900;
+    font-size: 22px;
```

Comment:
A font mixin should be used here instead. Use one from here: https://github.com/guardian/frontend/blob/master/static/src/stylesheets/pasteup/typography/_src.scss